ipfs-retriever/wrangler.toml

```diff
@@ -0,0 +1,59 @@
+name = "filbeam-ipfs-retriever"
+main = "bin/ipfs-retriever.js"
+compatibility_date = "2024-12-05"
+compatibility_flags = ["nodejs_compat"]
+logpush = true
+
+[[d1_databases]]
+binding = "DB"
+database_name = "test-db"
+database_id = "8cc92155-16f6-426a-b782-2965e0daf100"
+
+[[kv_namespaces]]
+binding = "BAD_BITS_KV"
+id = "2f2e5486ea0c48e993f6dff87a4aa102"
+
+[env.dev.vars]
+ENVIRONMENT = "dev"
+ORIGIN_CACHE_TTL = 86400
+CLIENT_CACHE_TTL = 31536000
+DNS_ROOT = ".localhost"
+
+[[env.dev.d1_databases]]
+binding = "DB"
+database_name = "dev-db"
+database_id = "8cc92155-16f6-426a-b782-2965e0daf101"
+
+[[env.dev.kv_namespaces]]
+binding = "BAD_BITS_KV"
+id = "2f2e5486ea0c48e993f6dff87a4aa102"
+
+[env.calibration.vars]
+ENVIRONMENT = "calibration "
+ORIGIN_CACHE_TTL = 86400
+CLIENT_CACHE_TTL = 31536000
+DNS_ROOT = ".ipfs.calibration.filbeam.io"
+
+[[env.calibration.d1_databases]]
+binding = "DB"
+database_name = "filcdn-calibration-db"
+database_id = "78f15bbb-391f-4797-9016-a6cb86c0b9b8"
+
+[[env.calibration.kv_namespaces]]
+binding = "BAD_BITS_KV"
+id = "178592ee0a3b4b00894a23186b3a0179"
+
+# [env.mainnet.vars]
+# ENVIRONMENT = "mainnet"
+# ORIGIN_CACHE_TTL = 86400
+# CLIENT_CACHE_TTL = 31536000
+# DNS_ROOT = ".ipfs.filbeam.io"
+
+# [[env.mainnet.d1_databases]]
+# binding = "DB"
+# database_name = "filcdn-mainnet-db"
+# database_id = "e8de6418-2cb7-4413-9ba0-a9c8aacf9a66"
+
+# [[env.mainnet.kv_namespaces]]
+# binding = "BAD_BITS_KV"
+# id = "7b03c39d53a041fdbe973c20285e16e9"
```

Comment:
IIRC, earlier this week, we decided to ship the IPFS retrieval worker to mainnet as well. Let's bring these mainnet wrangler config sections back.